nat-lab/tests/telio.py

```diff
@@ -1153,6 +1161,27 @@ async def save_logs(self) -> None:
                 f.write("\n\n\n\n--- SYSTEM LOG ---\n\n")
                 f.write(system_log_content)
 
+    async def save_moose_db(self) -> None:
+        """
+        Check if any the moose db files exists ("*-events.db"),
+        rename them to "str(test_name) + "_" + original_filename, and save them to "./logs",
+        delete the original file.
+        """
+        if os.environ.get("NATLAB_SAVE_LOGS") is None:
+            return
+
+        log_dir = "logs"
+        os.makedirs(log_dir, exist_ok=True)
+
+        test_name = test_name_safe_for_file_name()
+
+        moose_db_files = glob.glob("./*-events.db", recursive=False)
```

Comment:
Arent moose DB files supposed to be inside of container? :thinking:

Reply:
Yes, and they are, with these filenames:
```
CONTAINER_EVENT_PATH = "/event.db"
CONTAINER_EVENT_BACKUP_PATH = "/event_backup.db"
```

But when downloaded to test runner, they are renamed to `{alpha,bet,gamma}-events.db`.

Reply:
Aren'nt they downloaded only if some assert is done one them? I wonder if it would not be better to copy them from all client containers at this point.

Reply:
They're downloaded with `get_moose_db_file` ... and we have the latest copy already downloaded to test runner, on which the `assert` _might_ fail, so there will be some meat for investigation.

Reply:
ok